src/Discord.Net.Core/Entities/Interactions/MessageComponents/ComponentBuilder.cs

```diff
@@ -1415,6 +1418,9 @@ public string Value
                     throw new ArgumentOutOfRangeException(nameof(value), $"Value must not be longer than {MaxLength ?? LargestMaxLength}.");
                 if (value?.Length < (MinLength ?? 0))
                     throw new ArgumentOutOfRangeException(nameof(value), $"Value must not be shorter than {MinLength}");
+                if (Style == TextInputStyle.Short && value?.Contains('\n') == true)
```

Comment:
```suggestion
                if (Style == TextInputStyle.Short && value?.Contains('\n'))
```
There's no need to compare a `bool` with a value

Reply:
Since value can be null the entire expression may return null therefore I do the check  `` == true`` to ignore the check if no value is set.

Reply:
Oh hmmm
Yeah, makes sense.